### PR TITLE
Support restify

### PR DESCRIPTION
**Pull request by mdreamfly**
支持 restify 框架

ex:
var restify = require('restify');
var server = restify.createServer({});
server.use(restify.acceptParser(server.acceptable));
server.use(restify.queryParser());
server.use(restify.bodyParser());
server.post('/', wechat(config.wx,function(req,res,next){
        console.log(req.weixin);
        res.send('success');
}));

### CODE REVIEW

File: lib/wechat.js [hunk 1]
```diff
@@ -119,8 +119,9 @@ var load = function (stream, callback) {
   // support content-type 'text/xml' using 'express-xml-bodyparser', which set raw xml string
   // to 'req.rawBody'(while latest body-parser no longer set req.rawBody), see
   // https://github.com/macedigital/express-xml-bodyparser/blob/master/lib/types/xml.js#L79
-  if (stream.rawBody) {
-    callback(null, stream.rawBody);
+  // support restify req.body
+  if (stream.rawBody||stream.body) {
```
Comment (JacksonTian) [inline lib/wechat.js]: var body = stream.rawBody || stream.body;
if (body) {

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (JacksonTian) [inline .idea/vcs.xml]: 这个文件不要提交

Comment (mdreamfly) [inline .idea/vcs.xml]: 嗯 好的

**Comment by JacksonTian** [comment]
。。。为嘛要close？

**Comment by mdreamfly** [comment]
奇怪了。。。我没close啊

File: .idea/vcs.xml (hunk 2 above)
Comment (leoliaolei) [inline .idea/vcs.xml]: `var body = stream.rawBody || stream.body` works for restify

**Pull request closed without merging**